packages/react/src/experimental/UnderlinePanels/UnderlinePanels.tsx

```diff
@@ -52,6 +52,11 @@ export type UnderlinePanelsProps = {
    * Loading state for all counters. It displays loading animation for individual counters until all are resolved. It is needed to prevent multiple layout shift.
    */
   loadingCounters?: boolean
+  /**
+   * There are cases where you may not want the horizontal padding on items,
+   * and panels to make the tabs look horizontally aligned with the content above and below it.
+   */
+  noInlineGap?: boolean
```

Comment:
```suggestion
  flush?: boolean
```
Would you mind renaming this prop to `flush` to match a [similar implementation](https://github.com/primer/react/pull/5540) over in ActionBar?